publications-2019.bib

```diff
@@ -312,21 +313,27 @@ @article{Wick2019OptimizationWN
   url={https://arxiv.org/abs/1910.03424}
 }
 
-@Article{NW19,
-  author =       {N. Noii and T. Wick},
-  title =        {A phase-field description for pressurized and non-isothermal
-                  propagating fractures},
-  note =      {submitted},
-  year =         {2019},
-  url =       {https://arxiv.org/abs/1904.00196}
+@article{Noii_2019,
+   title={A phase-field description for pressurized and non-isothermal propagating fractures},
+   volume={351},
+   ISSN={0045-7825},
+   url={http://dx.doi.org/10.1016/j.cma.2019.03.058},
+   DOI={10.1016/j.cma.2019.03.058},
+   journal={Computer Methods in Applied Mechanics and Engineering},
+   publisher={Elsevier BV},
+   author={Noii, Nima and Wick, Thomas},
+   year={2019},
+   month={Jul},
+   pages={860–890}
```

Comment:
There's a unicode character hidden here. Can you change the dash to a regular minus?